sdk/js/src/Request.ts

```diff
@@ -79,12 +79,13 @@ export const getConfigJson = async (
     user: DVCPopulatedUser,
     logger: DVCLogger,
     options?: DVCOptions,
-    extraParams?: {sse: boolean, lastModified?: number}
+    extraParams?: { sse: boolean, lastModified?: number, etag?: string }
 ): Promise<BucketedUserConfig> => {
     const edgeDBParam = options?.enableEdgeDB ? ('&enableEdgeDB=' + options.enableEdgeDB): ''
     const sseParam = extraParams?.sse ? '&sse=1' : ''
     const lastModified = extraParams?.lastModified ? `&sseLastModified=${extraParams.lastModified}` : ''
-    const queryParams = `${serializeUser(user)}${edgeDBParam}${sseParam}${lastModified}`
+    const etag = extraParams?.etag ? `&sseEtag=${extraParams.etag}` : ''
```

Comment:
any chance this ends up not being URL safe?

Reply:
I don't think so, maybe with the `W/` ones? https://developer.mozilla.org/en-US/docs/Web/HTTP/Headers/ETag

Reply:
yeah that would probably break it.. you should probably URL encode this just to be safe